CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - Support disabling schema tests, and configuring tests from `dbt_project.yml` ([#3252](https://github.com/fishtown-analytics/dbt/issues/3252),
 [#3253](https://github.com/fishtown-analytics/dbt/issues/3253), [#3257](https://github.com/fishtown-analytics/dbt/pull/3257))
 - Add Jinja tag for tests ([#1173](https://github.com/fishtown-analytics/dbt/issues/1173), [#3261](https://github.com/fishtown-analytics/dbt/pull/3261))
+- Support detecting schema changes on incremental models per [#1132](https://github.com/fishtown-analytics/dbt/issues/1132)
```

Comment:
Nit: we like to include both the issue # and the PR # in the changelog entry
```suggestion
- Support detecting schema changes on incremental models ([#1132](https://github.com/fishtown-analytics/dbt/issues/1132), [#3288](https://github.com/fishtown-analytics/dbt/issues/3288))
```
Please also add yourself to the list of contributors below! It's well deserved :)

Reply:
10-4, thanks for the commentary.